foundry/foundry.py

```diff
@@ -87,7 +92,16 @@ def __init__(
             ].access_token,
         }
 
-    def load(self, name, download=True, globus=True, verbose=False, **kwargs):
+    def load(
```

Comment:
I believe this is not PEP 8 compliant https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
This is the Python Black autoformatting. It's ok to accept this for now, but I will look for a friendlier code formatter